tests/tests/swfs/avm2/sound_rootless/test.toml

```diff
@@ -0,0 +1,5 @@
+num_frames = 1
+ignore = true # enable once flash.media.Sound can load from file:// URL
```

Comment:
AFAIK You can fix this by replacing this:
```

            var sound_url:URLRequest = new URLRequest("file://noise.mp3");
            var sound:Sound = new Sound(sound_url);
```
with this:
```

            var sound_url:URLRequest = new URLRequest("noise.mp3");
            var sound:Sound = new Sound();
            sound.load(sound_url);
```

Reply:
changed the test as described. didn't play the sound/
for some reason i am getting the following when running with `RUST_LOG=warn,ruffle_core=debug,avm_trace=trace`:
```
2023-06-24T22:30:20.176283Z  INFO run_frame:run_all_phases_avm2: avm_trace: Initial state
2023-06-24T22:30:20.176327Z  INFO run_frame:run_all_phases_avm2: avm_trace: Removed root
2023-06-24T22:30:20.176389Z  WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.media.Sound constructor with arguments
2023-06-24T22:30:20.176415Z  WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.media.Sound.load() with context
2023-06-24T22:30:20.176437Z  INFO run_frame:run_all_phases_avm2: avm_trace: Loaded sound
2023-06-24T22:30:20.176493Z  INFO run_frame:run_all_phases_avm2: avm_trace: Playing sound
2023-06-24T22:30:20.176505Z  INFO run_frame:run_all_phases_avm2: avm_trace: [object SoundChannel]
```
despite the fact that `Sound` isn't getting an argument and `load` isn't getting a context

Reply:
`Encountered stub: AVM2 flash.media.Sound constructor with arguments`
You did something else wrong, in that case. Can you commit your changes so I can review?

Reply:
sent the commit with the above changes

Reply:
Your SWF file does not match up with your AS code

Reply:
removed the test ignore